layers/meta-balena-topic/recipes-kernel/linux/linux-xlnx/0001-ASoc-adi-Kconfig-Remove-depends-on-for-ADI-reference.patch

```diff
@@ -0,0 +1,28 @@
+From 91c0364479b932f473f72671c140d566dc758891 Mon Sep 17 00:00:00 2001
```

Comment:
Shouldn't all these patches be in the BSP layer instead, the meta-topic submodule?

Reply:
The machine (lcbzu9) that has been added consists of a Topic SoM module on top of the client's custom carrier board.
It is therefore technically not part of the Topic PCBs and not supported by meta-topic.

If you don't think these fit in meta-balena-topic, a workable suggestion might be to add something like "meta-lowpad".
Then I would move everything required to support the lowpad carrier board, including these patches, the machine.conf, fpga & device tree there.

Do you agree?

Reply:
I would rather we move all these to a meta-lowpad layer.
But if you are in a hurry now, we can merge them like this and you can move all lcbzu9 code at a later time in the new layer.